biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -1020,12 +1020,14 @@ private void doIncludes(File ubase, Properties p) {
 							} finally {
 								tmp.delete();
 							}
-						} catch (Exception mue) {
-							// ignore
+						} catch (MalformedURLException mue) {
+							if (fileMustExist)
+								error("Included file %s %s", file,
+										(file.isDirectory() ? "is directory" : "does not exist"));
+						} catch (Exception e) {
```

Comment:
Sure, I realise that, but I thought having a different error message for that catch encoded enough information to be worth including. I'd be happy to submit another PR without it if preferred?